activiti-spring-security-policies/src/test/java/org/activiti/spring/security/policies/RuntimeTestConfiguraiton.java

```diff
@@ -17,6 +19,7 @@
 public class RuntimeTestConfiguraiton {
 
     @Bean
+    @Primary
```

Comment:
@pancudaniel7 so... you don't need both beans.. ExtendedInMemoryUserDetailsManager is a UserDetailsService right? So just use the extended one.. and add users to the extended one.